dbt_platform_helper/commands/environment.py

```diff
@@ -134,6 +138,60 @@ def online(app, env):
         raise click.Abort
 
 
+@environment.command()
+def update_vpc_config():
+    """Update or add VPC configuration in copilot environment manifest files."""
+
+    ensure_cwd_is_repo_root()
+    session = get_aws_session_or_abort()
+    envs = list_copilot_local_environments()
+    vpcs = session.client("ec2").describe_vpcs()["Vpcs"]
+
+    for vpc in vpcs:
+        # skip unnamed VPCs
+        if "Tags" not in vpc.keys() or not [
+            tag["Value"] for tag in vpc["Tags"] if tag["Key"] == "Name"
+        ]:
+            continue
+
+        # expects string format "accountname-envname"
+        vpc_name = [tag["Value"] for tag in vpc["Tags"] if tag["Key"] == "Name"][0]
+        env_name = vpc_name.split("-")[-1]
+
+        if env_name in envs:
+            yuamel = ruamel.yaml.YAML(typ="rt")
+            click.echo(
+                f"\n>>> Updating {env_name} environment manifest.yml with current VPC and subnet ids\n"
+            )
+            env_manifest_path = Path(f"copilot/environments/{env_name}/manifest.yml")
+
+            with open(env_manifest_path, "r") as manifest_file:
+                config = yuamel.load(manifest_file)
+                new_data = config.copy()
+
+            subnets = session.client("ec2").describe_subnets(
+                Filters=[{"Name": "vpc-id", "Values": [vpc["VpcId"]]}]
+            )["Subnets"]
+            public_tag = {"Key": "subnet_type", "Value": "public"}
+            public = [
+                {"id": subnet["SubnetId"]} for subnet in subnets if public_tag in subnet["Tags"]
+            ]
+            private_tag = {"Key": "subnet_type", "Value": "private"}
+            private = [
+                {"id": subnet["SubnetId"]} for subnet in subnets if private_tag in subnet["Tags"]
+            ]
+            new_data["network"] = {
+                "vpc": {"id": vpc["VpcId"], "subnets": {"public": public, "private": private}}
+            }
```

Comment:
Part 2 of little refactor suggestion
```suggestion
            
            new_data["network"] = {
                "vpc": {
                    "id": vpc["VpcId"],
                    "subnets": {
                        "public": (filter_subnets(subnets, "public")),
                        "private": (filter_subnets(subnets, "private")),
                    },
                }
            }
```